examples/app_placement/app_v2.py

```diff
@@ -0,0 +1,53 @@
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from uvicorn import run
+
+from lightning import CloudCompute, LightningApp, LightningFlow, LightningWork
+
+
+class Work(LightningWork):
+    def __init__(self, **kwargs):
+        super().__init__(parallel=True, **kwargs)
+
+    def run(self):
+        fastapi_service = FastAPI()
+
+        fastapi_service.add_middleware(
+            CORSMiddleware,
+            allow_origins=["*"],
+            allow_credentials=True,
+            allow_methods=["*"],
+            allow_headers=["*"],
+        )
+
+        @fastapi_service.get(f"/{self.name}")
+        def get_root_name():
+            return {"Hello Word!"}
+
+        @fastapi_service.get("/")
+        def get_root():
+            return {"Hello Word!"}
+
+        run(fastapi_service, host=self.host, port=self.port)
+
+
+class Flow(LightningFlow):
+    def __init__(self):
+        super().__init__()
+        self.cloud_compute = CloudCompute(name="cpu-small")
```

Comment:
May be we shouldn't show this in the example - adding cloud_compute as an object attribute?

Reply:
I think this is fine. Why won't you expect CloudCompute to be attached to the flow ?